packages/web/src/pages/audio-rewards-page/WalletModal.tsx

```diff
@@ -166,16 +174,23 @@ const ModalContent = ({
     useSelector(getAccountBalance) ?? stringWeiToBN('0' as StringWei)
   const account = useSelector(getAccountUser)
   const amountPendingTransfer = useSelector(getSendData)
+  const { audiusSdk } = useAudiusQueryContext()
+
+  const { value: solWallet } = useAsync(async () => {
+    if (!account?.erc_wallet) return null
+    const sdk = await audiusSdk()
```

Comment:
wondering if doing these things here creates any noticeable latency with things opening up? if we have the user bank statically computed, this would be a bit faster. maybe that's the right compromise. wdyt?

Reply:
That's a good callout. I think if the promise for sdk is already resolved (which it should always be by the time we get here), this is pretty quick. The derive call also isn't making any network requests, so fairly fast.